test/powershell/Modules/Microsoft.PowerShell.Management/New-Item.Tests.ps1

```diff
@@ -181,4 +181,22 @@ Describe "New-Item with links" -Tags @('CI', 'RequireAdminOnWindows') {
             $_.FullyQualifiedErrorId | Should Be "NewItemSymbolicLinkElevationRequired,Microsoft.PowerShell.Commands.NewItemCommand"
         }
     }
+
+    It "New-Item -ItemType SymbolicLink should understand directory path ending with slash" {
+        $folderName = [System.IO.Path]::GetRandomFileName()
+        try
+        {            
+            $symbolicLinkPath = New-Item -ItemType SymbolicLink -Path "$tmpDirectory/$folderName/" -Value "/bar/"
+            $symbolicLinkPath | Should Not Be $null
+        }
+        finally
+        {
+            $Directoy = Get-Item "$tmpDirectory/$folderName/" -ErrorAction SilentlyContinue
```

Comment:
if these are all created in $TESTDRIVE, there's no clean up needed at all, as pester will take care of it
Also, please fix spelling error `Directoy` => `Directory`

---
In reply to: [110466085](https://github.com/PowerShell/PowerShell/pull/2974#discussion_r110466085) [](ancestors = 110466085)